backend/src/competitions/models/swimmer_models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from django.core.validators import MaxValueValidator, MinValueValidator
+
+from .shared_dictionaries import NATION
+
+GENDER = (
+    ('F', 'female'),
+    ('M', 'male'),
+)
+
+CLUB_TYPE = (
+    ('C', 'club'),
+    ('N', 'national team'),
+    ('R', 'regional team'),
+    ('U', 'unattached'),
+)
+
+
+class Athlete(models.Model):
+    first_name = models.CharField(max_length=16)
+    last_name = models.CharField(max_length=16)  # TODO: what's the max length? (case: double last name)
+    license = models.CharField(max_length=32, null=True)  # TODO: what's the max_length?
+    birth_date = models.DateField()
+    gender = models.CharField(max_length=1, choices=GENDER)
+    nation = models.CharField(max_length=2, choices=NATION, null=True)  # TODO
+    swrid = models.IntegerField(null=True, validators=[  # TODO
+        MinValueValidator(0),
+    ])
+
+
+class Club(models.Model):
+    short_name = models.CharField(max_length=20)
+    long_name = models.CharField(max_length=64, null=True)  # TODO: what's the max_length?
+    code = models.CharField(max_length=16, null=True)  # TODO: what_s the max length?
+    nation = models.CharField(max_length=2, choices=NATION, null=True)
```

Comment:
reformat fields in all models to look like this:
```suggestion
    nation = models.CharField(
        max_length=2, 
        choices=NATION, 
        null=True,
    )
```

Reply:
do we want to write all fields in this convention, or only the ones with multiple parameters?